executor/asyncloaddata/util.go

```diff
@@ -333,10 +377,11 @@ func GetJobInfo(
 		table_name,
 		import_mode,
 		progress,
-		create_user
+		create_user,
+		create_time
 		FROM mysql.load_data_jobs
-		WHERE job_id = %?;`,
-		OfflineThresholdInSec, jobID)
+		WHERE job_id = %? AND create_user = %?;`,
```

Comment:
user can only show jobs created by him/her?

Reply:
yes, to avoid the database user guess a jobID and see other database user's job

Reply:
then even `root` user cannot see all jobs

Reply:
Currently root user can query load_data_jobs table to bypass this check. Maybe we can ask PM to explain how to identify a privileged user and provide friendly API